src/blueprints/tv2/value-objects/tv2-layers.ts

```diff
@@ -25,7 +28,12 @@ export enum Tv2AtemLayer {
   LOOKAHEAD = 'atem_aux_lookahead',
   DOWNSTREAM_KEYER = 'atem_dsk',
   DVE = 'atem_dve',
-  DVE_BOXES = 'atem_dve_boxes'
+  DVE_BOXES = 'atem_dve_boxes',
+}
+
+export enum Tv2VideoMixerLayer {
+  AR = 'aux_ar',
```

Comment:
What is AR?

Reply:
Augmented Reality? Else no clue. This is what it is configured as...
![image](https://github.com/tv2/sofie-server/assets/23049647/7ecd4610-c3c1-4af5-96dc-2815fc554712)

Reply:
I think you are right. But do we use it? I don't think we integrate to zero density or unreal.
Perhaps we should call it `AUGMENTED_REALITY_AUXILIARY`, and then change it if we become wiser.

Reply:
I think you are right. But do we use it? I don't think we integrate to zero density or unreal.
Perhaps we should call it `AUGMENTED_REALITY_AUXILIARY`, and then change it if we become wiser.

Reply:
I think they might be used.
If Jacob is correct, then it's used to display the REPLAY feed to the commentators

Reply:
We could change it to `AC` and then we could call it `AUXILIARY_CLINTON` ;)